src/plugins/workspace/public/components/delete_workspace_modal/delete_details_modal.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiSmallButton,
+  EuiModal,
+  EuiModalBody,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCommentList,
+  EuiCommentProps,
+  EuiBadge,
+  EuiModalFooter,
+} from '@elastic/eui';
+import { CoreStart, WorkspaceAttribute } from 'opensearch-dashboards/public';
+
+export interface DeleteDetailsModalProps {
+  selectedWorkspaces: WorkspaceAttribute[];
+  failedWorksapces: WorkspaceAttribute[];
+  openModal: (
+    node: React.ReactNode,
+    options?: Parameters<CoreStart['overlays']['openModal']>['1']
+  ) => ReturnType<CoreStart['overlays']['openModal']>;
+}
+
+export const DeleteDetailsModal = (
```

Comment:
I'm prefer naming to `showDeleteDetailsModal`. Current naming `DeleteDetailsModal` more like a component, actually, it's an util function for now.